### PR TITLE
Normalize tabs to spaces

### PR DESCRIPTION
**Pull request by arcnmx**

### CODE REVIEW

File: src/sys/statfs.rs [hunk 1]
```diff
@@ -3,114 +3,114 @@ use errno::Errno;
 use std::os::unix::io::AsRawFd;
 
 pub mod vfs {
-	#[cfg(target_pointer_width = "32")]
+    #[cfg(target_pointer_width = "32")]
         pub mod hwdep {
```
Comment (kamalmarhubi) [inline src/sys/statfs.rs]: outdent this line by 4 spaces

**Comment by kamalmarhubi** [comment]
This looks good modulo, I can merge it after you fix the one niggle :-)

**Comment by arcnmx** [comment]
Change made!

**Comment by kamalmarhubi** [comment]
Merged :-)

**Pull request merged**